src/Services/GTT/shared/GTT.Infrastructure/Repositories/GroupRepository.cs

```diff
@@ -57,5 +60,92 @@ FROM Groups
                 throw new Exception(error);
             }
         }
+
+        public async Task<GroupResponse> UpdateGroup(UpdateGroupRequestModel group)
+        {
+            try
+            {
+                var checkGroup = $"SELECT * FROM Groups WHERE GroupId = {group.GroupId}";
+                var queryGroup = await _connection.QueryFirstOrDefaultAsync<GroupResponse>(checkGroup, commandType: CommandType.Text);
+
+                if (queryGroup == null)
+                {
+                    throw new Exception("Group is invalid !");
+                }
+
+                var queryName = $"SELECT * FROM Groups WHERE GroupName LIKE '%{group.GroupName}%'";
```

Comment:
I think We should choose a specific field for better optimization instead of select all